frontend/routes/app/app.component.tsx

```diff
@@ -161,6 +178,7 @@ export class App extends React.PureComponent<IProps, IState> {
 			<>
 				<DialogContainer />
 				<SnackbarContainer />
+				<LiveChat license={10338577} />
```

Comment:
What do you think about moving this license number to a new constant, which will contain main config of app?

Are you sure about adding an only single license number? Shouldn't be a license per environment (at least local and production)?
I don't know how exactly livechat works (we use Intercom), but - if you want to get some conversations' stats - this solution can obscure them.
Somebody can send some fake/test messages locally or on staging, which will make the stats corrupted.
If it's not that case, just omit this comment. ;)

Reply:
I think this should go through Client Config like most other things?

Reply:
I think this should go through Client Config like most other things?

Reply:
Sounds good